### DISCUSSION TITLE
/claimslist nicknames

### DISCUSSION THREAD

**Discussion by Azrian0** [GriefPrevention/GriefPrevention#2184] [Ideas]
Quite simple request.
The ability to nickname coordinates in /claimslist 
The claimslist can be sorted by numbers so if i had a claim in the end at let's say 0 0 0 and it was position 10 in my /claimslist I could do this:
/claimrename 10 endportal 

It would help identify claims if you are a player like myself and have tons of claims.

**Comment by Jikoo** [comment]
[ClaimClassifier](https://github.com/GPAddons/ClaimClassifier) can rename claims.
